e2e/framework/setup.go

```diff
@@ -0,0 +1,20 @@
+package framework
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+var (
+	kubeconfig    = flag.String("kubeconfig", "../hack/quickstart/cluster/auth/kubeconfig", "absolute path to the kubeconfig file")
+	expectedNodes = flag.Int("nodes", 0, "the number of nodes to expect")
```

Comment:
It will hard-exit on 0. I figured if I default to a bad value it will make sure people use the flag correctly. If we change the default to something useful maybe it should be 2 because I think that will be the default for most tests. (which also means terraform-quickstart should have the same default as well)